Khan/perseus
# Turn off interactivity when Interactive Graph in hint mode

## Summary:
Hint mode allows content creators to copy graphs directly from content, paste them into a hint, and have it automatically detect that the graph should be rendered as a hint. This work removes interactivity from the graph once it has been rendered as a hint.

(Need to confirm that adding a new graph into the hint lets you modify the graph.)

Issue: LEMS-2054

## Test plan:
- Run `yarn storybook` after checking out the branch associated with this PR
- Confirm the Hints Renderer graphs are not interactive (one story showing all graphs and also one story for each type): http://localhost:6006/?path=/story/perseus-renderers-hints-renderer--with-all-interactive-graphs

For further testing:
- Go to the Editor Page Demo: http://localhost:6006/?path=/story/perseuseditor-editorpage--demo
- Add an interactive graph widget to the Question box (try linear for now since it has start coords)
- Copy and paste that into the Hint input box
- Confirm with the preview that the hint graph is not interactive with the mouse or the keyboard and that the start coords change how it looks
- In the same way, confirm the other graph types are not interactive with either the mouse or the keyboard
